src/main/java/com/fabriccommunity/spookytime/world/feature/LargeSkeletalTreeFeature.java

```diff
@@ -0,0 +1,87 @@
+package com.fabriccommunity.spookytime.world.feature;
+
+import com.fabriccommunity.spookytime.registry.SpookyBlocks;
+import com.mojang.datafixers.Dynamic;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.Blocks;
+import net.minecraft.block.PillarBlock;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Direction;
+import net.minecraft.util.math.MutableIntBoundingBox;
+import net.minecraft.world.ModifiableTestableWorld;
+import net.minecraft.world.TestableWorld;
+import net.minecraft.world.gen.feature.AbstractTreeFeature;
+import net.minecraft.world.gen.feature.DefaultFeatureConfig;
+
+import java.util.Iterator;
+import java.util.Random;
+import java.util.Set;
+import java.util.function.Function;
+
+/**
+ * @author Indigo Amann
+ */
+public class LargeSkeletalTreeFeature extends AbstractTreeFeature<DefaultFeatureConfig> {
+    private static final BlockState LOG = Blocks.BONE_BLOCK.getDefaultState();
+
+    public LargeSkeletalTreeFeature(Function<Dynamic<?>, ? extends DefaultFeatureConfig> function, boolean boolean1) {
+        super(function, boolean1);
+    }
+
+    private void addLog(Set<BlockPos> set, ModifiableTestableWorld modifiableTestableWorld, BlockPos blockPos, Direction.Axis axis, MutableIntBoundingBox mutableIntBoundingBox) {
```

Comment:
Lacking `@Override`s 'round these lands I believe.

Reply:
Tree Features methods cannot be overriden from my experience, this is something we can't really help here.

Reply:
Nope